### PR TITLE
Feature/cobolcheck

### PR DESCRIPTION
**Pull request by 0xE282B0**
Hi, I changed the PR from @axtens to do the following:

- `fetch-cobolcheck` was created according to `fetch-configlet`
  - binary is downloaded for now from: https://github.com/0xE282B0/cobol-check/releases/tag/v0.1.0-graal
  - when it is merged upstream we can switch to https://github.com/openmainframeproject/cobol-check
- test configuration has been moved from `hello-world` test to `./etc`
- `test.sh` has been created to execute the test

@axtens @KTSnowy What do you think about the test layout?

### CODE REVIEW

**Review by KTSnowy — APPROVED** [review]
Looks good to me. We'll be using gnuCOBOL right?

**Comment by 0xE282B0** [comment]
@KTSnowy,
Cobolcheck is independent of the compiler, it can be set in the scripts like [etc/scripts/linux_gnucobol_run_tests](https://github.com/exercism/cobol/pull/5/files#diff-bd8f71d9612fba2d20e200a529d4bbcf599301d541a137de0dcf1db372287fe2R8).

But yes, the default is gnuCOBOL.

**Comment by iHiD** [comment]
(cc @ErikSchierboom)

File: exercises/practice/hello-world/test.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+if [ ! -f ../../../bin/cobolcheck ]; then
+    echo "Cobolcheck not found, try to fetch it."
+    DIR=$(pwd)
+    cd ../../../bin/
+    ./fetch-cobolcheck
+    cd $DIR
+fi
+../../../bin/cobolcheck -c ../../../etc/config.properties -p hello-world
```
Comment (ErikSchierboom) [inline exercises/practice/hello-world/test.sh:9]: Unfortunately, this relative structure won't work as students only receive the files with the exercise's directory. This means that any files the student needs have to be duplicated within the exercise directory.

A nice way to verify this is by joing the cobol track (which I've just added to the database) by going to https://exercism.org/tracks/cobol (the track is only visible to maintainers and not listed in the tracks list). Once joined, you can use the CLI to download the exercise locally (e.g. `exercism download --exercise=hello-world --track=cobol`) to inspect if things would work for students.

**Pull request merged**